langchain_tools_demo/main.py

```diff
@@ -50,34 +72,24 @@ async def lifespan(app: FastAPI):
     asyncio.gather(*close_client_tasks)
 
 
-# FastAPI setup
-app = FastAPI(lifespan=lifespan)
-app.mount("/static", StaticFiles(directory="static"), name="static")
-# TODO: set secret_key for production
-app.add_middleware(SessionMiddleware, secret_key="SECRET_KEY")
-templates = Jinja2Templates(directory="templates")
-BASE_HISTORY: list[BaseMessage] = [
-    AIMessage(content="I am an SFO Airport Assistant, ready to assist you.")
-]
-CLIENT_ID = os.getenv("CLIENT_ID")
-
-
-@app.route("/", methods=["GET", "POST"])
+@routes.get("/")
+@routes.post("/")
 async def index(request: Request):
     """Render the default template."""
     # Agent setup
     agent = await get_agent(request.session, user_id_token=None)
+    templates = Jinja2Templates(directory="templates")
```

Comment:
nit: I think it's okay for this to be declared globally. It's essentially a static property